frontend/src/component/viewer.vue

```diff
@@ -247,6 +248,25 @@ export default {
       this.idleTimer = false;
       this.hasTouch = false;
 
+      // Add Safari-specific space key handler
+      if (this.isSafari) {
+        lightbox.on('bindEvents', () => {
+          lightbox.pswp.element.addEventListener('keydown', (e) => {
+            if (e.code === 'Space' || e.key === ' ') {
```

Comment:
From my point of view it is better to use the `keyCode` to check the necessary element, what do you think? 
In this case we can add the comment to this line, for example: 32 - Space, 13 - Enter, docs - https://www.toptal.com/developers/keycode